### PR TITLE
Update Boilerplate and Myth

### PR DESCRIPTION
**Pull request by RobLoach**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,16 +1,8 @@
...
- * Copyright (c) 2015 Charlike Mike Reagent, contributors.
- * Released under the MIT license.
- */
-
 'use strict';
 
 var myth = require('myth');
 
 exports.name = 'myth';
-exports.inputFormats = ['css', 'myth'];
```
Comment (TimothyGu) [inline index.js:13]: Why?

File: package.json [hunk 2]
```diff
@@ -1,47 +1,30 @@
...
-  "description": "Myth support for JS Transformers",
-  "scripts": {
-    "lint": "jshint index.js && jscs index.js --reporter inline",
-    "test": "node test",
-    "test-cov": "istanbul cover test",
-    "test-travis": "istanbul cover test --report lcovonly"
+  "version": "1.0.0",
+  "description": "Myth support for JSTransformers.",
+  "dependencies": {
+    "myth": "~1.5.0"
```
Comment (TimothyGu) [inline package.json]: Why not `^1.5.0`?

File: index.js (hunk 1 above)
Comment (RobLoach) [inline index.js:13]: Oops

File: package.json (hunk 2 above)
Comment (RobLoach) [inline package.json]: :+1:

**Comment by tunnckoCore** [comment]
cuz i dont love semver very much and there are a lot of crazy guys in the wild that dont follow it and the shit raise. For me its better to bump versions after reasonable review and after few weeks or months (depend on their release cycle)

**Comment by RobLoach** [comment]
> For me its better to bump versions after reasonable review and after few weeks or months

Feel free to review reasonably after a few weeks or months! :+1:

**Pull request merged**